Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -612,8 +636,13 @@ protected override void
 
       switch (op) {
         case BinaryExpr.ResolvedOpcode.Add:
-          opString = "+";
-          break;
+          opString = "+"; break;
+
+        case BinaryExpr.ResolvedOpcode.Mul:
+          opString = "*"; break;
+
+        case BinaryExpr.ResolvedOpcode.EqCommon:
+          opString = "=="; break;
```

Comment:
Not necessarily this PR, but next time could you consider moving this to a switch expression that returns a nullable string, and if the string returned is null, it calls the default case? It might simplify all these case and breaks.

Reply:
That would be nice for the moment, but I have every reason to expect that well will need to set one of the other out variables (L618-L625) eventually.